docs/offboarding.rst

```diff
@@ -166,42 +166,39 @@ was in effect.
 On the Secure Viewing Station
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. First, change the UID of the current SecureDrop submission key to avoid
-   mixing up the old and new keys.
-
-   From the *Secure Viewing Station* Applications Menu, choose **Utilities ▸
-   Passwords and Keys**, and select the SecureDrop Application Key from the
+#. From the *Secure Viewing Station* Applications Menu, choose **Accessories ▸
+   Kleopatra**, and select the SecureDrop Application Key from the
    list of available keys.
 
 
    |select securedrop key|
+   
+#. From the details view that appears, click the *Add email address* button.
 
+   |key details|
+   
+#. In the dialog window that appears, choose *Advanced*.
 
-#. Double-click the key, and in the **Names and Signature** tab, add a name
-   such as "OLD <Your Organization> SecureDrop Submission Key - Do Not Delete
-   - Retired <Date>". (This is a local-only change to stop you from mixing up
-   the old and new keys).
-
+   |add email|
 
+#. Change the name of the key to "OLD <Your Organization> SecureDrop Submission
+   Key - Do Not Delete", and set the comment to "Retired <Date of Retirement>".
+   
```

Comment:
From my testing, this adds a UID rather than changing it (though it pre-populates the fields with information from the existing UID). I also don't see a better option in Kleopatra, and it seems to do the job in setting the key apart. 

I'd suggest a slight reword for clarity:

> Set the name field to "OLD <Your Organization> SecureDrop Submission  Key - Do Not Delete", and set the comment field to "Retired `<Date of Retirement>`". Click **OK** to add this information to the key.